lightningd/jsonrpc.c

```diff
@@ -135,7 +135,8 @@ static void log_to_json(unsigned int skipped,
 		else
 			json_add_string(info->response, "direction", "OUT");
 
-		json_add_hex(info->response, "data", log+1, tal_count(log)-1);
+		if (tal_count(log) > 0)
```

Comment:
Line 133 reads `if (log[0])` implying that `tal_count(log)` must be at least 1, assuming that `log` is a tal-allocated array.